.github/workflows/test.yml

```diff
@@ -13,13 +13,19 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
+        node-version: [18, 20]
 
     steps:
       - uses: actions/checkout@v4.1.7
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v5.1.1
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Set up Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ matrix.node-version }}
+        run: npm ci
```

Comment:
@lana-w this needs to be written as a separate step: see https://github.com/david-yz-liu/memory-viz/blob/master/.github/workflows/ci.yml#L21-L22